### PR TITLE
[APP-7307]: Remove logo file path manipulation in CLI for OrganizationSetLogo.

### PR DESCRIPTION
**Pull request by maxhorowitz**
https://viam.atlassian.net/browse/APP-7307

### CODE REVIEW

**Comment by maxhorowitz** [comment]
Validated it works:
```
maxhorowitz at stingray in ~/dev/rdk on APP-7307 [$]
$ go run cli/viam/main.go organization logo set --org-id="********" --logo-path="../../Desktop/jeep.png"
Error: Logo file is too large: 217947 bytes (max size is 200KB)
exit status 1
```

File: cli/client.go [hunk 1]
```diff
@@ -345,15 +344,7 @@ func (c *viamClient) organizationLogoSetAction(cCtx *cli.Context, orgID, logoFil
 	if err := c.ensureLoggedIn(); err != nil {
 		return err
 	}
-
-	// determine whether this is a valid file path on the local system
-	logoFilePath = strings.ToLower(filepath.Clean(logoFilePath))
```
Comment (jr22) [inline cli/client.go:350]: can we keep the filepath.Clean?

File: cli/client.go [hunk 2]
```diff
@@ -345,15 +344,7 @@ func (c *viamClient) organizationLogoSetAction(cCtx *cli.Context, orgID, logoFil
...
-
-	if len(logoFilePath) < 5 || logoFilePath[len(logoFilePath)-4:] != ".png" {
-		return errors.Errorf("%s is not a valid .png file path", logoFilePath)
```
Comment (jr22) [inline cli/client.go:353]: why remove? because it might be a png but not have png ending?

Comment (jr22) [reply cli/client.go:353]: we do this checking server side so agree we dont necessarily need it here

File: cli/client.go (hunk 1 above)
Comment (maxhorowitz) [reply cli/client.go:350]: Yes

File: cli/client.go [hunk 3]
```diff
@@ -345,15 +344,7 @@ func (c *viamClient) organizationLogoSetAction(cCtx *cli.Context, orgID, logoFil
...
-	}
-
-	logoFile, err := os.Open(logoFilePath)
+	logoFile, err := os.Open(logoFilePath) //nolint:gosec
```
[the remark this answers is not in the document]
Comment (maxhorowitz) [reply cli/client.go:347]: If I add `filepath.Clean` maybe it won't be necessary - but yeah really didn't know what / why needed

**Review by jr22 — APPROVED** [review]

**Review by njooma — APPROVED** [review]

**Pull request merged**